<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HashCalc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="hashEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>341</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="verifyButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>105</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Verify!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>141</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Elige el archivo</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboHash">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
